### ISSUE TITLE
Is this package deprecated? Because in Vuetify since 3.4.0 we have Data Tables

### ISSUE THREAD

**Issue by seyfer** [HC200ok/vue3-easy-data-table#353]
https://vuetifyjs.com/en/components/data-tables/introduction

The question is - is this package going to be supported, or it is over?

**Comment by douglasg14b** [comment]
Why would it be deprecated because vuetify, a different library, has data tables? Prime-vue has data tables as well...

As a vuetiy user I cam here because veutify data tables are nearly unconstomizeable at every level.

**Comment by seyfer** [comment]
@douglasg14b the purpose of this library being created is because Vuetify had no Data Table component after the update from v2 to v3. The author of this library here closed this gap. But now Vuetify 3 has this Data Table component.
Therefore my question has a reason. 

just read https://hc200ok.github.io/vue3-easy-data-table-doc/introduction.html#why

and seems like this library is not updating anymore...

**Comment by douglasg14b** [comment]
Gotcha. Yeah it doesn't seem to get updates anymore. 

However, Vuetify 3 data tables really are not that great or ergonomic to use right now. Customizing and controlling them in non-trivial ways is a nightmare. This lib archives baseline customizability that Vuetify ones are miles from right now.

It might even be worth forking this library, refactoring it, and making some bugfixes to re-release it under a new name & maintainer.

**Comment by DubG1** [comment]
i agree i came here because the vuetify data table has so limited customization and this one covers almost everything i need except a few things so i would be really happy if this library would get continued

**Comment by brunotutrut** [comment]
@HC200ok hey, this project is dead?
